iwyu.cc

```diff
@@ -2426,9 +2427,16 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
         // parse it to '<new>' before using, so any path that does
         // that, and is clearly a c++ path, is fine; its exact
         // contents don't matter that much.
+
         const FileEntry* use_file = CurrentFileEntry();
-        preprocessor_info().FileInfoFor(use_file)->ReportFullSymbolUse(
-            CurrentLoc(), "<new>", "operator new");
+        const clang::DirectoryLookup* TmpDir = nullptr;
```

Comment:
We use `lowercase_with_underscores` naming for variables in IWYU, not LLVM's `PascalCase`.

This can probably be called `curdir` to signify what role it plays in the argument list.

Reply:
used `curdir` as a variable name